assets/src/edit-story/app/story/useStoryReducer/reducers/combineElements.js

```diff
@@ -87,7 +91,7 @@ function combineElements(state, { firstElement, secondId }) {
     'scale',
     'focalX',
     'focalY',
-    'backgroundOverlay',
+    'flip',
```

Comment:
Why is `flip` added? I know @miina did something in #5960 here, maybe it just needs merging with main?

Reply:
This came from a rebase I did off of main. I'm not sure why it's being highlighted here as being added. I'll cross reference @miina 's PR and make sure it's supposed to be there

Reply:
It's because @miina [removed it](https://github.com/google/web-stories-wp/pull/5960/files#diff-dda4bb6f48bf8eff39ec9d57b481f2847a5bcf0e7229507374cf01074024cd30L90) in #5960 - you didn't remove it in your merge resolution, so it now appears as being re-added. Please check with @miina if it's correct to just remove it again here.

Reply:
Yes, it should be removed since the flip would come from the element that's being dragged into and not the element being dragged. E.g. dragging into a flipped shape would keep the shape flipped.

Reply:
Just FI: there's an open issue to clarify the flipping logic as well and it might bring some changes but for now the `'flip'` should be removed: https://github.com/google/web-stories-wp/issues/5966

Reply:
Awesome. thanks! must of got mixed up when rebasing. will remove rn